src/test/java/dk/cngroup/lentils/repository/StatusRepositoryTest.java

```diff
@@ -99,6 +103,17 @@ public void updateStatusTest() {
         assertEquals(SOLVED, status1.getCypherStatus());
     }
 
+    @Test
```

Comment:
Tenhle test jestli dobře vidím by možná bylo lepší dát do nějaké třídy StatusServiceTest, než do RepositoryTest.

Reply:
taky mne to napadlo, ale v tride StatusServiceTest se StatusRepository mockuje,
takze jsem to dal sem, abych se presvedcil, ze se opravdu inicializuje status v db

Reply:
presunuto do StatusServiceIntegrationTest